web/styles/right_sidebar.css

```diff
@@ -93,7 +110,13 @@ $user_status_emoji_width: 24px;
     }
 
     .empty-list-message {
-        font-size: 1.2em;
+        font-style: italic;
+        color: var(--color-text-empty-list-message);
+        /* Overwrite default empty list font size, to look better under the subheaders. */
+        font-size: 14px;
+        /* Override .empty-list-message !important styling */
+        padding: 0 !important;
```

Comment:
I don't understand this comment -- are you saying this `!important` overrides another `!important` somewhere else?

Reply:
Ahh, yeah. I suspect the base `empty-list-message` base CSS class doesn't super make sense; can you start a #frontend discussion about how we should refactor that?

Reply:
https://chat.zulip.org/#narrow/stream/6-frontend/topic/empty-list-message.20styling